lizmap/modules/admin/controllers/theme.classic.php

```diff
@@ -257,7 +294,7 @@ public function removeThemeImage()
         }
 
         // update theme
-        $modifytheme = $theme->update($data);
```

Comment:
Yep, but the `$theme->update($data)` should stay, so I did not remove it

Reply:
Yes, I understood. I was talking about the variable, not the function call.
Instead of
```php
$modifyTheme = $theme->update($data);
```
it could be
```php
$theme->update($data);
```

;-)

Feel free to skip